internal/container/client.go

```diff
@@ -482,6 +502,9 @@ func (cl *Client) Resolve(ctx context.Context, name string) (Spec, error) {
 
 	spec := NewSpec(cl.Target, ids.Manifest, ids.Config)
 	spec.TLSVerify = cl.GetTLSVerify()
+	if ids.ListManifest != "" {
+		spec.ListDigest = ids.ListManifest.String()
+	}
```

Comment:
You're right, we don't need to check if it's empty.  We could also add the `TLSVerify` value to the constructor since we always set it like that as well.